### PR TITLE
Create molecularEntity Terminology

### PR DESCRIPTION
**Pull request by lzehl**
related to #83 

Include steps:
1) create controlledTerms schema
2) add molecularEntity to terminology list
3) add first set of instances for molecularEntity

@UlrikeS91 let me know if I missed something.

### CODE REVIEW

**Comment by lzehl** [comment]
@UlrikeS91 first set of instances (as well as the schema) for molecular entities are ready for review :)

File: instances/molecularEntity/GABA-A_BZ.jsonld [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/GABA-A/BZ",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "The GABA-A/BZ is a distinct binding site for benzodiazepines that is situated at the interface between the \u03b1- and \u03b3-subunits of \u03b1- and \u03b3-subunit containing GABA-A receptors.",
```
Comment (UlrikeS91) [inline instances/molecularEntity/GABA-A_BZ.jsonld:7]: " [...] \u03b1- and \u03b3-subunits of \u03b1- and \u03b3-subunit containing [...]" 

Is this on purpose? It sounds a bit odd and the subunits generally are pretty cryptic. Any way to revise this?

File: instances/molecularEntity/alpha-1Receptor.jsonld [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/alpha-1Receptor",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "The alpha-1 receptor is a subclass of the  adrenoceptor group that bind epinephrine or norepinephrine as endogenous ligands.",
```
Comment (UlrikeS91) [inline instances/molecularEntity/alpha-1Receptor.jsonld]: there is an extra space between "the" and "adrenoceptor"

File: instances/molecularEntity/alpha-1Receptor.jsonld [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "interlexIdentifier": null,
+    "knowledgeSpaceLink": null,
+    "name": "alpha-1 receptor",
+    "preferredOntologyIdentifier": null,
+    "synonym": [
+        "alpha1",
+        "alpha-1 adrenergic receptor",
+        "alpha 1",
+        "\u03b11 receptor",
+        "\u03b11 adrenergic receptor"
```
Comment (UlrikeS91) [inline instances/molecularEntity/alpha-1Receptor.jsonld:18]: same comment as above. rather cryptic, but I guess it is correct?

File: instances/molecularEntity/alpha-2Receptor.jsonld [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "interlexIdentifier": null,
+    "knowledgeSpaceLink": null,
+    "name": "alpha-2 receptor",
+    "preferredOntologyIdentifier": null,
+    "synonym": [
+        "alpha2",
+        "alpha-2 adrenergic receptor",
+        "alpha 2",
+        "\u03b12 receptor",
+        "\u03b12 adrenergic receptor"
```
Comment (UlrikeS91) [inline instances/molecularEntity/alpha-2Receptor.jsonld:18]: same comment as above. rather cryptic, but I guess it is correct?

File: instances/molecularEntity/alpha-2Receptor.jsonld [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/alpha-2Receptor",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "The alpha-2 receptor is a subclass of the  adrenoceptor group that bind epinephrine or norepinephrine as endogenous ligands.",
```
Comment (UlrikeS91) [inline instances/molecularEntity/alpha-2Receptor.jsonld]: also has the extra space

File: instances/molecularEntity/alpha-4Beta-2Receptor.jsonld [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    "interlexIdentifier": "http://uri.interlex.org/ilx_0597802",
+    "knowledgeSpaceLink": null,
+    "name": "alpha-4 beta-2 receptor",
+    "preferredOntologyIdentifier": "http://id.nlm.nih.gov/mesh/2018/M0356600",
+    "synonym": [
+        "nicotinic acetylcholine alpha4beta2 receptor",
+        "alpha-4 beta-2 nicotinic receptor",
+        "alpha-4 beta-2 receptor",
+        "nicotinic receptor alpha4beta2",
+        "\u03b14\u03b22 receptor"
```
Comment (UlrikeS91) [inline instances/molecularEntity/alpha-4Beta-2Receptor.jsonld:18]: same comment as above. rather cryptic, but I guess it is correct?

File: instances/molecularEntity/anterogradeTracer.jsonld [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/anterogradeTracer",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "Molecule that is taken up by the cell bodies of neurons and transported towards the axon terminals, used for anterograde tract tracing studies in the nervous system.",
+    "description": null,
+    "interlexIdentifier": null,
+    "knowledgeSpaceLink": "https://knowledge-space.org/wiki/NLXMOL:1012002#anterograde-tracer",
+    "name": "anterograde tracer",
+    "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/NLXMOL_1012002",
+    "synonym": []
```
Comment (UlrikeS91) [inline instances/molecularEntity/anterogradeTracer.jsonld]: needs to be "null" if no synonyms are listed

File: instances/molecularEntity/anterogradeTracer.jsonld [hunk 8]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/anterogradeTracer",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "Molecule that is taken up by the cell bodies of neurons and transported towards the axon terminals, used for anterograde tract tracing studies in the nervous system.",
```
Comment (UlrikeS91) [inline instances/molecularEntity/anterogradeTracer.jsonld]: I would not say "cell bodies". This is typically the main place, but not exclusively. Maybe:

"Molecule that is taken up by neurons (e.g., by viral transfection mechanisms, by other cell internalization mechanisms or passive diffusion) and transported towards the axon terminals, used for anterograde tract tracing studies in the nervous system."

File: instances/molecularEntity/biomarker.jsonld [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/biomarker",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "A substance used as an indicator of a biological state, most commonly disease.",
+    "description": null,
+    "interlexIdentifier": "http://uri.interlex.org/ilx_0101294",
+    "knowledgeSpaceLink": null,
+    "name": "biomarker",
+    "preferredOntologyIdentifier": "http://uri.neuinfo.org/nif/nifstd/nlx_mol_20090517",
+    "synonym": []
```
Comment (UlrikeS91) [inline instances/molecularEntity/biomarker.jsonld]: needs to be "null" if no synonyms are listed

File: instances/molecularEntity/biotinylatedDextranAmine.jsonld [hunk 10]
```diff
@@ -0,0 +1,19 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/biotinylatedDextranAmine",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "A 'biotinylated dextran amine' is an organic compound which is used as an anterograde and retrograde neuroanatomical tracers.",
```
Comment (UlrikeS91) [inline instances/molecularEntity/biotinylatedDextranAmine.jsonld]: "[...] as an [...] tracer." or "[...] as [...] tracers." (I would prefer singular "tracer")

File: instances/molecularEntity/calciumCalmodulinProteinKinaseIi.jsonld [hunk 11]
```diff
@@ -0,0 +1,18 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/calciumCalmodulinProteinKinaseIi",
```
Comment (UlrikeS91) [inline instances/molecularEntity/calciumCalmodulinProteinKinaseIi.jsonld]: calciumCalmodulinProteinKinaseIi --> calciumCalmodulinProteinKinaseI**I** (Kinase II, the lower case second i is wrong)

File: instances/molecularEntity/calciumCalmodulinProteinKinaseIi.jsonld [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "The 'calcium calmodulin protein kinase II' is a protein with a core domain architecture consisting of a Protein kinase domain and a C-terminal Calcium/calmodulin dependent protein kinase II Association domain.",
```
Comment (UlrikeS91) [inline instances/molecularEntity/calciumCalmodulinProteinKinaseIi.jsonld]: "[...] of a Protein kinase domain a [...]" -- protein lowercase?

File: instances/molecularEntity/calciumCalmodulinProteinKinaseIiAlphaChain.jsonld [hunk 13]
```diff
@@ -0,0 +1,25 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/calciumCalmodulinProteinKinaseIiAlphaChain",
```
Comment (UlrikeS91) [inline instances/molecularEntity/calciumCalmodulinProteinKinaseIiAlphaChain.jsonld]: the Kinase II i again

File: instances/molecularEntity/fluorescentMicrospheres.jsonld [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/fluorescentMicrospheres",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "Fluorescent microspheres are non-toxic, non-biologically reactive small polymers embedded with fluorescent dye which are used in medical imaging, as markers for fluorescent microscopy and as standards for flow cytometry fluorescent cell sorting.",
+    "description": null,
+    "interlexIdentifier": null,
+    "knowledgeSpaceLink": null,
+    "name": "fluorescent microspheres",
+    "preferredOntologyIdentifier": null,
+    "synonym": []
```
Comment (UlrikeS91) [inline instances/molecularEntity/fluorescentMicrospheres.jsonld]: needs to be "null" if no synonyms are listed

File: instances/molecularEntity/growthFactor.jsonld [hunk 15]
```diff
@@ -0,0 +1,16 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/growthFactor",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "The \"growth factor\" comprises signal molecules that are involved in the control of cell growth and differentiation.",
```
Comment (UlrikeS91) [inline instances/molecularEntity/growthFactor.jsonld]: maybe just make the " into ' around 'growth factor'?

File: instances/molecularEntity/intrabody.jsonld [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/intrabody",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "An \"intrabody\" is an antibody that works within the cell to bind an intracellular protein.",
+    "description": null,
+    "interlexIdentifier": null,
+    "knowledgeSpaceLink": null,
+    "name": "intrabody",
+    "preferredOntologyIdentifier": null,
+    "synonym": []
```
Comment (UlrikeS91) [inline instances/molecularEntity/intrabody.jsonld]: needs to be "null" if no synonyms are listed

File: instances/molecularEntity/intrabody.jsonld [hunk 17]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/intrabody",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "An \"intrabody\" is an antibody that works within the cell to bind an intracellular protein.",
```
Comment (UlrikeS91) [inline instances/molecularEntity/intrabody.jsonld]: maybe just make the " into ' around 'intrabody'?

File: instances/molecularEntity/medetomidine.jsonld [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/medetomidine",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "Medetomidine is a synthetic drug used as both a surgical anesthetic and analgesic.",
+    "description": null,
+    "interlexIdentifier": "http://uri.interlex.org/ilx_0488544",
+    "knowledgeSpaceLink": "https://knowledge-space.org/wiki/CHEBI:48552#medetomidine",
+    "name": "medetomidine",
+    "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/CHEBI_48552",
+    "synonym": []
```
Comment (UlrikeS91) [inline instances/molecularEntity/medetomidine.jsonld]: needs to be "null" if no synonyms are listed

File: instances/molecularEntity/neurobiotin.jsonld [hunk 19]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    "@id": "https://openminds.ebrains.eu/instances/molecularEntity/neurobiotin",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+    "definition": "Neurobiotin is a biotin derivative with moleular weight 286 kDa that can be used as an anterograde and retrograde tracer in the nervous system.",
+    "description": null,
+    "interlexIdentifier": "http://uri.interlex.org/ilx_0107453",
+    "knowledgeSpaceLink": "https://knowledge-space.org/wiki/NLXMOL:1012015#neurobiotin",
+    "name": "neurobiotin",
+    "preferredOntologyIdentifier": "http://uri.neuinfo.org/nif/nifstd/nlx_157299",
+    "synonym": []
```
Comment (UlrikeS91) [inline instances/molecularEntity/neurobiotin.jsonld]: needs to be "null" if no synonyms are listed

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/molecularEntity/GABA-A_BZ.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/molecularEntity/GABA-A_BZ.jsonld:7]: @olinux how should we deal with special characters ? (especially in molecular entities we will have a need for them for correct names)

File: instances/molecularEntity/anterogradeTracer.jsonld (hunk 8 above)
Comment (lzehl) [reply instances/molecularEntity/anterogradeTracer.jsonld]: see revised definition

File: instances/molecularEntity/GABA-A_BZ.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/molecularEntity/GABA-A_BZ.jsonld:7]: @olinux confirmed it should be interpreted correctly

File: instances/molecularEntity/intrabody.jsonld (hunk 16 above)
Comment (lzehl) [reply instances/molecularEntity/intrabody.jsonld]: should be null now

File: instances/molecularEntity/intrabody.jsonld (hunk 17 above)
Comment (lzehl) [reply instances/molecularEntity/intrabody.jsonld]: done

File: instances/molecularEntity/medetomidine.jsonld (hunk 18 above)
Comment (lzehl) [reply instances/molecularEntity/medetomidine.jsonld]: done

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**